### PR TITLE
Enable arc warden + some requested changes while i was at it

### PR DESCRIPTION
**Pull request by Honeth**
allows people to start play testing arc warden because he still needs testing.

he wasn't enabled due to outdated herolist

### CODE REVIEW

**Comment by Chronophylos** [comment]
If you're at it go to `npc_heroes_custom.txt` and fix the typo in elder titan aka elser titan

File: game/scripts/vscripts/items/refresher_core.lua [hunk 1]
```diff
@@ -61,5 +61,9 @@ function item_refresher_core:OnSpellStart()
   end
 end
 
+function item_refresher_core:IsRefreshable()
+return false
```
Comment (Chronophylos) [inline game/scripts/vscripts/items/refresher_core.lua]: now you need to indent it correctly :P

Comment (Honeth) [inline game/scripts/vscripts/items/refresher_core.lua]: really now you want to to indent ffs XD will do

Comment (Chronophylos) [inline game/scripts/vscripts/items/refresher_core.lua]: :D

**Pull request merged**